### PR TITLE
Creeper mode

### PR DESCRIPTION
**Pull request by hell0w0rld2**
Cuts driveMotorsMultiplier in half when left bumper is pressed on the drive controller.

### CODE REVIEW

File: robot.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def createObjects(self):
         self.instantiateSubsystemGroup("solenoids", solenoidFactory)
 
         # Check each componet for compatibility
-        testComponentCompatibility(self, ShooterLogic)
+        #testComponentCompatibility(self, ShooterLogic)
```
Comment (ChrisHirsch) [inline robot.py]: This shouldn't have been commented out and committed

File: components/driveTrain.py [hunk 2]
```diff
@@ -46,6 +48,14 @@ def setArcade(self, speed, rotation):
         self.arcadeSpeed = speed
         self.arcadeRotation = rotation
 
+    def creeperMode(self):
+        """set driveMotorMultiplier to .25"""
+        self.prevMultiplier = self.driveMotorsMultiplier
+        self.driveMotorsMultiplier = .25
```
[the remark this answers is not in the document]
Comment (hell0w0rld2) [reply components/driveTrain.py]: creates a better comment in creeperMode()

File: components/driveTrain.py [hunk 3]
```diff
@@ -46,6 +48,14 @@ def setArcade(self, speed, rotation):
...
+
+    def disableCreeperMode(self):
+        self.driveMotorsMultiplier = self.prevMultiplier
```
[the remark this answers is not in the document]
Comment (hell0w0rld2) [reply components/driveTrain.py]: comment created in disableCreeperMode()

File: robot.py (hunk 1 above)
Comment (hell0w0rld2) [reply robot.py]: uncommented that line

**Comment by ChrisHirsch** [comment]
This PR fails in autonomous mode
  File "Y:\Users\chirsch\src\raptacon\Robot-2020\venv\lib\site-packages\magicbot\magicrobot.py", line 371, in autonomous
    self._automodes.run(
  File "Y:\Users\chirsch\src\raptacon\Robot-2020\venv\lib\site-packages\robotpy_ext\autonomous\selector.py", line 271, in run
    on_exception()
  File "Y:\Users\chirsch\src\raptacon\Robot-2020\venv\lib\site-packages\robotpy_ext\autonomous\selector.py", line 269, in run
    self._on_iteration(timer.get())
  File "Y:\Users\chirsch\src\raptacon\Robot-2020\venv\lib\site-packages\robotpy_ext\autonomous\selector.py", line 337, in _on_iteration
    self.active_mode.on_iteration(time_elapsed)
  File "Y:\Users\chirsch\src\raptacon\Robot-2020\venv\lib\site-packages\magicbot\state_machine.py", line 649, in on_iteration
    self.execute()
  File "Y:\Users\chirsch\src\raptacon\Robot-2020\venv\lib\site-packages\magicbot\state_machine.py", line 610, in execute
    state.run(self, tm, tm - state.start_time, initial_call)
  File "<string>", line 1, in <lambda>
  File "Y:\Users\chirsch\src\raptacon\Robot-2020\autonomous\autonomous.py", line 35, in turn
    self.driveTrain.setup(-self.drive_speed, self.drive_speed)
TypeError: setup() takes 1 positional argument but 3 were given

**Comment by ChrisHirsch** [comment]
> This PR fails in autonomous mode
> File "Y:\Users\chirsch\src\raptacon\Robot-2020\venv\lib\site-packages\magicbot\magicrobot.py", line 371, in autonomous
> self._automodes.run(
> File "Y:\Users\chirsch\src\raptacon\Robot-2020\venv\lib\site-packages\robotpy_ext\autonomous\selector.py", line 271, in run
> on_exception()
> File "Y:\Users\chirsch\src\raptacon\Robot-2020\venv\lib\site-packages\robotpy_ext\autonomous\selector.py", line 269, in run
> self._on_iteration(timer.get())
> File "Y:\Users\chirsch\src\raptacon\Robot-2020\venv\lib\site-packages\robotpy_ext\autonomous\selector.py", line 337, in _on_iteration
> self.active_mode.on_iteration(time_elapsed)
> File "Y:\Users\chirsch\src\raptacon\Robot-2020\venv\lib\site-packages\magicbot\state_machine.py", line 649, in on_iteration
> self.execute()
> File "Y:\Users\chirsch\src\raptacon\Robot-2020\venv\lib\site-packages\magicbot\state_machine.py", line 610, in execute
> state.run(self, tm, tm - state.start_time, initial_call)
> File "", line 1, in
> File "Y:\Users\chirsch\src\raptacon\Robot-2020\autonomous\autonomous.py", line 35, in turn
> self.driveTrain.setup(-self.drive_speed, self.drive_speed)
> TypeError: setup() takes 1 positional argument but 3 were given

Actually it turns out that it was a problem in master.

**Pull request merged**